### PR TITLE
Update Weight Information for Pixhawk4mini

### PR DESCRIPTION
**Pull request by bys1123**

### CODE REVIEW

File: en/flight_controller/pixhawk4_mini.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ The *Pixhawk<sup>&reg;</sup> 4 Mini* autopilot is designed for engineers and hob
   * Servo Rail Input: 0~24V
   * Max current sensing: 120A
 * Weight and Dimensions:
+  * Weight: 37.2g
```
Comment (hamishwillee) [inline en/flight_controller/pixhawk4_mini.md:36]: Thanks! How sure are you of this value?
I can't find anything official, but there are other values quoted publically https://www.getfpv.com/holybro-pixhawk-4-mini-w-pm06-v2.html

Comment (bys1123) [reply en/flight_controller/pixhawk4_mini.md:36]: Holybro updated their pdf datasheet.

**Comment by bys1123** [comment]
[Pixhawk4mini_technical_data_sheet.pdf](https://github.com/PX4/px4_user_guide/files/4434625/Pixhawk4mini_technical_data_sheet.pdf)

**Comment by hamishwillee** [comment]
Thanks very much. I'll update the datasheet too!

**Pull request merged**